decisions/wc_religion_decisions.txt

```diff
@@ -259,6 +260,47 @@ decisions = {
 			factor = 0.25
 		}
 	}
+	
+	candle_ritual = {
```

Comment:
It doesn't give much.
I would like to see traits adding like in vanilla. You go to Mecca because you know you may get useful traits.

Reply:
Well, I could add that later through random list